source/getting-started/managing-dependencies.md

```diff
@@ -0,0 +1,141 @@
+Ember CLI uses the [Bower](http://bower.io) package manager, making it easy
+to keep your front-end dependencies up to date. NPM is used for development
+dependencies (like addons).
```

Comment:
How does the user know when to look for an Ember addon versus a Bower package? I think we need to be more specific here, something like:

As you're developing your Ember app, you'll likely run into common scenarios that aren't addressed by Ember itself, such as authentication or using SASS for your stylesheets. Ember CLI provides a common format called **addons** for distributing reusable libraries to solve these problems. Additionally, you may want to make use of front-end dependencies like a CSS framework on a JavaScript datepicker that aren't specific to Ember apps. Ember CLI supports installing these packages through the standard [Bower](http://bower.io) package manager.